scripts/ci/check-namespace

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+import subprocess
+import os
+
+def check_file(file_path, namespce):
```

Comment:
Typo? `namespce` -- also, it's set in L13?

Reply:
Good catch... damn.